server.go

```diff
@@ -3786,6 +3780,11 @@ func (s *server) peerConnected(conn net.Conn, connReq *connmgr.ConnReq,
 		}
 	}
 
+	var towerClient wtclient.TowerClientManager
```

Comment:
nit: use `s.towerClientMgr` directly as it's also either `nil` or not?

Reply:
yeah that is what I originally did but ran into this gotcha: https://go.dev/play/p/QXeX8Abqfug

Reply:
which then causes a panic if `s.towerClientMgr` is used directly

Reply:
Ah, I see. Always forget about that interface gotcha... Might be worth a comment though?

Reply:
good idea - added :)